src/content/docs/logs/ui-data/timestamp-support.mdx

```diff
@@ -0,0 +1,200 @@
+
+The data type will be:
+- A number, for Unix epoch seconds or milliseconds timestamp
+- A string, for ISO8601 formatted timestamp
+
+Other considerations:
+- If the timestamp field is not present or incorrectly specified (for example, with an invalid format), the message will be timestamped using the ingest time.
+- Timestamps are converted to Unix epoch milliseconds and stored internally as a long data type, with the reserved keyword `timestamp`.
+- Payloads with timestamps older than 48 hours will be dropped.
+
+Depending on the JSON body format, ​the timestamp field can be allocated in four different places:
+- Inside the [simplified set of attributes](https://docs.newrelic.com/docs/logs/log-api/introduction-log-api/#simple-json) of the JSON body message when sending a single JSON object.
+- Inside the [common](https://docs.newrelic.com/docs/logs/log-api/introduction-log-api/#json-common) object in the detailed set of attributes of the JSON body message when sending one or more JSON objects. The timestamp will apply to all log messages of this JSON.
+- Inside each log message in the [logs](https://docs.newrelic.com/docs/logs/log-api/introduction-log-api/#json-logs) object in the detailed set of attributes of the JSON body message when sending one or more JSON objects. The timestamps will only apply to that log message.
+- In addition, the timestamp can be indicated in the “message” JSON field when it is a valid JSON message. Our log management capabilities will parse any message attribute as JSON. The resulting JSON attributes in the parsed message will be added to the log. (link) 
+
+Below are some examples of JSON logs with a valid `timestamp` attribute that will override the ingest `timestamp`:
+
+1. If we receive a JSON log like this with `timestamp` in **epoch** format (with no ms precision):
+
+```
+{ 
+    "timestamp":1701445572,
+    "message":"This is my sample JSON log message with timestamp attribute in epoch format"
+}
+```
+
+We will override the ingest `timestamp` with the epoch `timestamp` contained in the log, and the result will look like this:
+
+```
+"timestamp" : 1701445572000,
+"message":"This is my sample JSON log message with timestamp attribute in epoch format"
+```
+
+2. Or, if we receive a JSON log like this with timestamp in **ISO8601** format like this:
+
+```
+{
+    "timestamp":"2023-12-01T15:46:26.607Z",
+    "message":"This is my sample JSON log message with timestamp attribute in ISO8601 format"
+}
+```
+
+We will override the ingest `timestamp` with the ISO8601 `timestamp` (converted to epoch) contained in the log, and the result will look like this:
+
+```
+"timestamp": 1701445586607,
+"message":"This is my sample JSON log message with timestamp attribute in ISO8601 format"
+```
+
+## Setting the log timestamp using Parsing rules [#timestamp_parsing_rules]
```

Comment:
I'm a little lost at this section. I'm not sure what Grok expressions are and how they relate to parsing rules. This section also feels like it could be broken out into two, too. One is more reference material with some examples of how to use Grok expressions (?) and the second is to set up parsing rules specific to timestamps. What is the connective tissue here? @marcelschlapfer